core/iwasm/fast-jit/fe/jit_emit_control.c

```diff
@@ -578,13 +578,12 @@ handle_op_else(JitCompContext *cc, uint8 **p_frame_ip)
 static bool
 handle_next_reachable_block(JitCompContext *cc, uint8 **p_frame_ip)
 {
-    JitBlock *block = cc->block_stack.block_list_end, *block_prev;
+    JitBlock *block = jit_block_stack_top(&cc->block_stack);
```

Comment:
Can you also modify "cc->block_stack.block_list_end" in other places to "jit_block_stack_top"?